### PR TITLE
Add config to call `CreateServiceTimeSeries`

### PR DESCRIPTION
**Pull request by jsuereth**
- Adds a binary flag where all metrics go to CreateServiceTimeSeries or not.
- Update metric descriptor logic so that *no* metric descriptors are sent if we're in `CreateServiceTimeSeries` mode.
- Tests using an integration test (as there are no unit tests for this portion of code, nor was I sure how to appropriately hijack).

### CODE REVIEW

**Comment by dashpole** [comment]
cc @laoj2

**Review by dashpole — APPROVED** [review]

**Comment by jsuereth** [comment]
@dashpole I found out we were missing the code to auto-detect ProjectID and update config when instantiating an exporter.   Added it, PTAL

File: exporter/collector/metricsexporter.go [hunk 1]
```diff
@@ -79,6 +81,22 @@ func newGoogleCloudMetricsExporter(
 ) (component.MetricsExporter, error) {
 	setVersionInUserAgent(cfg, set.BuildInfo.Version)
 
+	// TODO - Share this lookup somewhere
+	if cfg.ProjectID == "" {
+		if ctx == nil {
```
Comment (dashpole) [inline exporter/collector/metricsexporter.go:86]: Checking for nil context is unusual in go.  Can we make sure the passed-in context is not nil everywhere and remove this check?

Comment (jsuereth) [reply exporter/collector/metricsexporter.go:86]: this was copy-pasta from opencensus exporter, I can just remove.

**Review by dashpole — APPROVED** [review]

**Pull request merged**